package.json

```diff
@@ -47,6 +48,7 @@
     "eslint-plugin-no-only-tests": "^2.4.0",
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-prettier": "^3.1.4",
+    "jsonpath": "^1.1.1",
```

Comment:
No need to add a dependency for this, we want to have as little dependencies as possible in lodestar. Just do a regex yourself :+1:

Reply:
hmmmm ok :+1: